### DISCUSSION TITLE
Unable to download manifests found in steambd.

### DISCUSSION THREAD

**Discussion by BMlight** [SteamRE/DepotDownloader#573] [Q&A]
### What did you expect to happen?

download the first version of the game.

### Instead of that, what actually happened?

The download was rejected.

### Which operating system are you running on?

Windows

### Version

2.7.4

### Command

.\DepotDownloader.exe -app 1932160 -depot 1932161 -manifest 3967462165400161672 -username [except] -password [except]

### Relevant log output

```shell
Connecting to Steam3... Done!
Logging '[except]' into Steam3...
STEAM GUARD! Use the Steam Mobile App to confirm your sign in...
 Done!
Using Steam3 suggested CellID: 8
Got 529 licenses for account!
Got AppInfo for 1932160
Using app branch: 'public'.
Got depot key for 1932161 result: OK
Processing depot 1932161
Downloading depot manifest... Got manifest request code for 1932161 3967462165400161672 result: 0
No manifest request code was returned for 1932161 3967462165400161672
Encountered 401 for depot manifest 1932161 3967462165400161672. Aborting.

Unable to download manifest 3967462165400161672 for depot 1932161
App 1932160 was not completely downloaded.
The operation was canceled.
Disconnected from Steam
```

### Additional Information
![no title](https://github.com/user-attachments/assets/e6880133-0981-4a91-bd5d-340cc623fd86)
https://steamdb.info/depot/1932161/manifests/

the manifest available on steamdb.

**Comment by ClaudeLib** [comment]
Same here when trying to download the 1.0.7.0 build of GTA IV. Trying to do `download_depot 12210 12211 164691614299318355` in Steam Console returns Access Denied, and `./DepotDownloader -app 12210 -depot 12211 -manifest 164691614299318355 -username -qr` returns 401. Downloading the more recent manifest 4406763129603688303 seems to work, for now. So it's not exactly an issue with DD, I wonder what it is.

EDIT. Oh, now I see it in the ReadMe, my bad.

**Reply by 2called-chaos** [reply]
I wouldn't be too sure actually about what the FAQ says. I am trying to download all versions of a beta that requires key invite and somewhere in the middle it stopped with 401 errors. But I can still download the first manifests so it's not the age, but here's the kicker. A fellow friend can still download them, it's seemingly just my account that gets 401's (but I can also still download the latest ones)

**Reply by ClaudeLib** [reply]
Well, that's interesting. If it's not too much to ask, could you please check if you or your friend can download `-app 12210 -depot 12211 -manifest 164691614299318355`?
